makefiles/info-global.inc.mk

```diff
@@ -109,6 +109,11 @@ info-boards-supported:
 info-boards-features-missing:
 	@for f in $(BOARDS_FEATURES_MISSING); do echo $${f}; done | column -t
 
+IS_BOARD_SUPPORTED ?= $(filter $(ENV_BOARD), $(BOARDS))
+.PHONY: is-board-supported
+is-board-supported:
+	$(Q)test -n "$(IS_BOARD_SUPPORTED)"
```

Comment:
`Q` is set after `GLOBAL_GOALS` handling so in this case `$(Q)` is never set to `@`. We can either move that handling up or use `@` directly here.

Reply:
well then `@` directly, that matches with the rest in this file